testing/sdk_metrics.py

```diff
@@ -24,6 +25,22 @@ def get_scheduler_metrics(service_name, timeout_seconds=15 * 60):
     return sdk_cmd.service_request("GET", service_name, "/v1/metrics").json()
 
 
+def get_failure_metrics(service_name: str) -> typing.Dict:
```

Comment:
If we change this to return the sum, we could change the function name.

Reply:
I'm also not really sure about having this in `sdk_metrics`. In our context metrics mean something different -- they are generated by the scheduler or the underlying framework. Here we are only interested in the number of tasks in a (failed) terminal state. I would argue that `sdk_tasks` is a better module to implement this in.